lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -225,14 +295,19 @@ def add_address(post, payment_method, address, options, address_type)
           postalCode:            address[:zip],
           country:               lookup_country_code(address[:country])&.value,
           email:                 options[:email].presence || 'null@cybersource.com',
-          phoneNumber:           address[:phone]
+          phoneNumber:           address[:phone],
           # merchantTaxID:         ship_to ? options[:merchant_tax_id] : nil,
-          # company:               address[:company],
+          company:               address[:company]
           # companyTaxID:          address[:companyTaxID],
```

Comment:
❓ Is this set of comments required for a further development?

Reply:
I'm not sure about it; I just removed the company committed because I needed it.